openupgrade_scripts/scripts/mail/17.0.1.15/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,488 @@
+mail         / ir.attachment            / voice_ids (one2many)          : NEW relation: discuss.voice.metadata
+mail         / ir.ui.view               / type (False)                  : selection_keys is now '['activity', 'calendar', 'form', 'gantt', 'graph', 'hierarchy', 'kanban', 'pivot', 'qweb', 'search', 'tree']' ('['activity', 'calendar', 'form', 'gantt', 'graph', 'kanban', 'pivot', 'qweb', 'search', 'tree']')
+# NOTHING TO DO: new feature
+
+mail         / mail.activity            / active (boolean)              : NEW hasdefault: default
+mail         / mail.activity            / attachment_ids (many2many)    : NEW relation: ir.attachment
+mail         / mail.activity            / date_done (date)              : NEW isfunction: function, stored
+mail         / mail.activity            / state (selection)             : selection_keys is now '['done', 'overdue', 'planned', 'today']' ('['overdue', 'planned', 'today']')
+mail         / mail.activity.plan       / active (boolean)              : NEW hasdefault: default
+mail         / mail.activity.plan       / company_id (many2one)         : NEW relation: res.company, hasdefault: default
+mail         / mail.activity.plan       / name (char)                   : NEW required
+mail         / mail.activity.plan       / res_model (selection)         : NEW required, selection_keys: function
+mail         / mail.activity.plan       / res_model_id (many2one)       : NEW relation: ir.model, required, hasdefault: compute
+mail         / mail.activity.plan       / template_ids (one2many)       : NEW relation: mail.activity.plan.template
+mail         / mail.activity.plan.template / activity_type_id (many2one)   : NEW relation: mail.activity.type, required, hasdefault: default
+mail         / mail.activity.plan.template / note (html)                   : NEW
+mail         / mail.activity.plan.template / plan_id (many2one)            : NEW relation: mail.activity.plan, required
+mail         / mail.activity.plan.template / responsible_id (many2one)     : NEW relation: res.users, hasdefault: compute
+mail         / mail.activity.plan.template / responsible_type (selection)  : NEW required, selection_keys: ['on_demand', 'other'], hasdefault: default
+mail         / mail.activity.plan.template / sequence (integer)            : NEW hasdefault: default
+mail         / mail.activity.plan.template / summary (char)                : NEW hasdefault: compute
+mail         / mail.activity.type       / keep_done (boolean)           : NEW
+# NOTHING TO DO: new feature
+
+mail         / mail.alias               / alias_full_name (char)        : NEW isfunction: function, stored
+# DONE pre and post migration: create column in pre and fill value in post
```

Comment:
Why on pre? I think it doesn't matter, and in post you save the column addition.

And you can group all the entries belonging to this.